### ISSUE TITLE
Cannot train each branch separately (large-object or medium-object branch)

### ISSUE THREAD

**Issue by tranleanh** [YunYang1994/tensorflow-yolov3#599]
The model works well until I want to train large-object branch only!

In training phase, I freeze medium-object branch and small-object branch (set "trainable=False"), and I only calculate loss using large-object branch loss value. However, after 2 epochs, training loss tends to be 0.

Could you please help me to understand this issue? Thanks in advance!
